stargeo/urls.py

```diff
@@ -75,4 +75,6 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^api/', include(router.urls)),
     url(r'^docs', SwaggerSchemaView.as_view()),
+
+    url(r'^statistics$', 'core.views.stats', name='statistics'),
```

Comment:
We already have `stats/` 
https://github.com/idrdex/star-django/blob/48901624f0b6126e16576797361a43df16325f28/stargeo/urls.py#L69